src/lib/components/workspace/pythonblocks/pythonBlocks.svelte

```diff
@@ -0,0 +1,116 @@
+<script lang="ts">
```

Comment:
this is almost entirely a duplicate of Blocks.svelte, it should just be in blocks.svelte with a single if condition around the generator